### DISCUSSION TITLE
How to pass empty parameter on Aptos explorer?

### DISCUSSION THREAD

**Discussion by luoyhang003** [aptos-labs/aptos-developer-discussions#496] [Questions] [answered]
### Describe your question in detail.

<img width="447" alt="image" src="https://github.com/user-attachments/assets/c0a9537c-b8e9-413d-b3a9-c10917f5e65f">

### Which operating system are you using?

macOS

### Which SDK or tool are you using? (if any)

N/A

**Comment by Alexzander-Stone** [comment, marked as answer]
Hey there @luoyhang003,

For an empty array on the Explorer, feel free to throw in:
`[]`

Here's an example:
![Screenshot 2024-10-17 at 5 12 02 PM](https://github.com/user-attachments/assets/6b43998c-294d-45e9-91cf-1b8626e92159)
